src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -287,7 +288,8 @@ private ConfigProperties() {
         JobCleaner.JOB_KEY,
         ManifestCleanerJob.JOB_KEY,
         UnmappedGuestEntitlementCleanerJob.JOB_KEY,
-        InactiveConsumerCleanerJob.JOB_KEY
+        InactiveConsumerCleanerJob.JOB_KEY,
+        RevokeEntitlementsJob.JOB_KEY
```

Comment:
This job shouldn't be on this list, since its scoped to an owner, while the list should only contain scheduled/global jobs instead

Reply:
Removed